### PR TITLE
Changelog for 5.0

### PR DESCRIPTION
**Pull request by TheoChevalier**
Date (tomorrow) is a tentative. But I thought we’d want to get that ready.

### CODE REVIEW

File: app/models/changelog_data.php [hunk 1]
```diff
@@ -56,9 +57,126 @@
...
+    '5.0' => [
+        [
+            'message' => ['“To Latin” button added to Serbian results, providing on-the-fly transliteration'],
+            'section' => ['end_user'],
+            'type'    => ['new'],
+            'commit'  => ['###TODO###'],
+            'authors' => ['SkySymbol'],
+        ],
+        [
+            'message' => ['Transvision look &amp; feel has been updated. It’s flatter, looks more modern and is focused on readability'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: improving readability?

File: app/models/changelog_data.php [hunk 2]
```diff
@@ -56,9 +57,126 @@
...
+        ],
+        [
+            'message' => ['A menu has been added on <a href="/channelcomparison">Channel Comparison</a> view to navigate through content'],
+            'section' => ['end_user'],
+            'type'    => ['better'],
+            'commit'  => ['4be64e6270046548ac6d8818fe6060a2923408bf'],
+            'authors' => ['tchevalier'],
+        ],
+        [
+            'message' => ['Same contextual info and actions (e.g. “File a bug“, string errors, copy to clipboard…) are now consistently available in both search results and entity views'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: `in results for string and entity searches` sounds more correct?

File: app/models/changelog_data.php [hunk 3]
```diff
@@ -56,9 +57,126 @@
...
+        ],
+        [
+            'message' => ['Transvision now tells whether a string is empty or missing'],
+            'section' => ['end_user'],
+            'type'    => ['better'],
+            'commit'  => ['64dda7a9256ff0537b360980b2756a0df03d4548'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Removed “button on hover” experiments from search results'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: experiment (sing.)

**Comment by TheoChevalier** [comment]
Note to self before I go to bed, I forgot to talk about the API change where missing strings are not returned anymore

**Comment by TheoChevalier** [comment]
;~
               ./|\.
             ./ /| `\.
            /  | |   `\.
           |   | |     `\.
           |    \|       `\.
         .  `----|__________\.
          \-----''----.....___
          \               ""/
      ~^~^~^~^~^`~^~^`^~^~^`^~^~^
       ~^~^~`~~^~^`~^~^~`~~^~^~

SHIP IT.

**Pull request merged**